ray-operator/DEVELOPMENT.md

```diff
@@ -116,4 +116,12 @@ make sync
 
 # Reproduce CI error for job "helm-chart-verify-rbac" (consistency 5)
 python3 ../scripts/rbac-check.py
-```
\ No newline at end of file
+```
+
+### Run end-to-end tests locally
+
+We have some [end-to-end tests](https://github.com/ray-project/kuberay/blob/master/.github/workflows/actions/compatibility/action.yaml) on GitHub Actions.
```

Comment:
I'd add a couple of sentences here:

`These tests operate small Ray clusters running within a [kind](https://kind.sigs.k8s.io/) (Kubernetes-in-docker) environment. To run the tests yourself, follow these steps:`

Reply:
Hmm, looks like running a test requires specifying a KubeRay operator SHA from a commit that's already merged to master?

How would you test your local development operator code? Maybe we can make the full operator image configurable via environment variable?

Reply:
Good idea. I will update it.

Reply:
Updated.